src/components/Endorsements.astro

```diff
@@ -0,0 +1,56 @@
+---
+// © 2024 Vlad-Stefan Harbuz <vlad@vladh.net>
+// SPDX-License-Identifier: Apache-2.0
+
+import Endorsement from "./Endorsement.astro";
+---
+
+<div class="endorsement-group">
+  <Endorsement
+    name="Johannes Schickling"
+    handle="@schickling"
+    imageUrl="/images/endorsers/schickling.webp"
+    sourceUrl="https://x.com/schickling/status/1832361798130954335"
+  >
+    <p>I can't overstate how excited I am about this. ❤️</p>
+  </Endorsement>
+  <Endorsement
+    name="Raymmar"
+    handle="@raymmar_"
+    imageUrl="/images/endorsers/raymmar.webp"
+    sourceUrl="https://x.com/raymmar_/status/1832464000996548806"
+  >
+    <p>Great initiative. [Vital] for the next step of internet growth. 💪</p>
```

Comment:
"Much needed and vital" didn't work because of word-wrapping, or ... ?

Reply:
Yeah the 💪🏻 emoji ended up being the only thing on the second line. I can try something else if you think the full sentence would be better.

Reply:
(Could just remove the emoji)

Reply:
Yeah let's do that